cfme/fixtures/pytest_selenium.py

```diff
@@ -982,7 +982,8 @@ def all_selected_options(self):
         Selenium's all_selected_options iterates over ALL of the options, this directly returns
         only those that are selected.
         """
-        return execute_script("return arguments[0].selectedOptions;", element(self))
+        return execute_script(
+            "return arguments[0] ? arguments[0].selectedOptions || [] : [];", element(self))
```

Comment:
I still hate the ternary:
`return (arguments[0] || {selectedOptions: []}).selectedOptions`?